programming/python/index.md

```diff
@@ -20,11 +20,10 @@ There are a number of tutorials out there which might help you to learn to progr
     there is a lot there and it may be a little overwhelming.
 
 * A number of tutorials for beginners are linked to from [here](http://wiki.python.org/moin/BeginnersGuide/NonProgrammers).
-    [Byte of Python](http://www.swaroopch.com/notes/Python_en:Functions) seems like a good one.
+    [Byte of Python](https://python.swaroopch.com/) seems like a good one.
```

Comment:
It looks like this guide now defaults to Python 3, which may be confusing for some. Maybe it's worth simply pointing at the existing "Whirlwind Tour" page we have (https://studentrobotics.org/docs/tutorials/python)?

Reply:
Thats weird it defaulted to python 2.7 for me, ill have another look tomorrow